packages/expo-updates/android/src/main/java/expo/modules/updates/statemachine/UpdatesStateContext.kt

```diff
@@ -77,4 +78,37 @@ data class UpdatesStateContext(
       result.putMap("context", contextMap)
       return result
     }
+
+  /**
+   * Creates a Bundle to be returned to JS on a call to nativeStateMachineContext()
+   */
+  val bundle: Bundle
```

Comment:
may be able to combine this with the `writeableMap` method above by using `Arguments.fromBundle()`:

```
  /**
   * Creates a WritableMap to be sent to JS on a state change.
   */
  val writableMap: WritableMap
    get() = Arguments.createMap().apply {
      putMap("context", Arguments.fromBundle(bundle) )
    }
```

Reply:
Very nice catch, saves a lot of duplicate code :)